SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -4724,46 +5470,47 @@ void CIncEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container,
       if (implicit)
         Jacobian.AddBlock(iPoint, iPoint, Jacobian_i);
 
-//      /*--- Viscous contribution, commented out because serious convergence problems ---*/
-//
+      /*--- Viscous contribution, commented out because serious convergence problems ---*/
+
 //      if (viscous) {
-//
-//        /*--- Set laminar and eddy viscosity at the inlet ---*/
-//
-//        V_inlet[nDim+3] = node[iPoint]->GetLaminarViscosity();
-//        V_inlet[nDim+4] = node[iPoint]->GetEddyViscosity();
+//        
+//        /*--- Set transport properties at the inlet ---*/
+//        
```

Comment:
Maybe we can remove that commented block ?

Reply:
I have actually decided to uncomment and keep these blocks. However, we should discuss the original change in the compressible code and turb models, which may have been made by @fpalacios? I find that the performance is ok with the incompressible solver, and that this might be important for internal flows or truncated domains.